cl_manycore/regression/cuda/cuda_tests.h

```diff
@@ -0,0 +1,31 @@
+#ifndef __CUDA_TESTS_H
+#define __CUDA_TESTS_H
+
+#ifdef COSIM
```

Comment:
Can you remove the ifdef COSIM and use the angle bracket includes for everything except the simulation libraries?

Reply:
Nice catch. Fixed it for all three cuda, library and spmd headers.